conf/functions_worker.yml

```diff
@@ -44,3 +44,4 @@ rescheduleTimeoutMs: 60000
 initialBrokerReconnectMaxRetries: 60
 assignmentWriteMaxRetries: 60
 instanceLivenessCheckFreqMs: 30000
+metricsSamplingPeriodSec: 60
```

Comment:
Should the default be zero?(That is let prometheus or other system collect the metrics)?

Reply:
if we run function in standalone cluster then one doesn't have prometheus so, default function should update the rate so, user can fetch metrics using cli.